data/available_datasets.json

```diff
@@ -1,6 +1,162 @@
 {
-    "last_updated": "2023-01-13",
+    "last_updated": "2023-02-01",
     "datasets": [
+        {
+            "doi": "10.1016/j.concog.2023.103464",
+            "available_items": [
+                {
+                    "type": "data",
+                    "repo_name": "OSF",
+                    "accession_number": "",
+                    "description": "Students survey data",
+                    "data_type": [
+                        "Social science and humanities data"
+                    ],
+                    "data_url": "https://osf.io/9stbm/"
+                }
+            ],
+            "title": "COVID-19 infection does not seem to affect cognition in college students",
+            "issue": "",
+            "volume": "108",
+            "publisher": "Elsevier BV",
+            "published": "2023-01-30",
+            "author": [
+                "Francis G.",
+                "Thunell E."
+            ]
+        },
+        {
+            "doi": "10.1038/s41597-022-01880-8",
+            "available_items": [
+                {
+                    "type": "data",
+                    "repo_name": "OSF",
+                    "accession_number": "",
+                    "description": "Familydemic survey data",
+                    "data_type": [
+                        "Social science and humanities data"
+                    ],
+                    "data_url": "https://osf.io/mjfz3/"
+                }
+            ],
+            "title": "Familydemic Cross Country and Gender Dataset on work and family outcomes during COVID-19 pandemic",
+            "issue": "1",
+            "volume": "10",
+            "publisher": "Springer Science and Business Media LLC",
+            "published": "2023-01-03",
+            "author": [
+                "Kurowska A.",
+                "Barardehi I. H.",
+                "Fuller S.",
+                "Petts R. J.",
+                "Kaufman G."
+            ]
+        },
+        {
+            "doi": "10.1038/s41586-022-05512-4",
+            "available_items": [
+                {
+                    "type": "data",
+                    "repo_name": "Zenodo",
+                    "accession_number": "",
+                    "description": "Randomized controlled trial data from Sweden and United States",
```

Comment:
Controlled trial data on what though? examining impact of financial incentives on vaccination? also - randomised with an S (British spelling, so no z)